README.md

```diff
@@ -17,39 +147,49 @@ git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-In the main `sparkle` folder, copy `.env.example` to a new file `.env.local`, and fill it with the appropriate details.
-
-You can read more about the various `.env` files that you can use at:
-
-- https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env
+### Step 2: Install Packages
 
 Install the platform dependencies with `npm`:
 
-(**Note:** `npm` v7+ is not supported, it will cause issues with our `package-lock.json`, and you may end up with the wrong dependency versions)
+**Important Note:** `npm` v7+ is not supported, it will cause issues with our `package-lock.json`, and you may end up with the wrong dependency versions. Check your current npm version with `npm -v`.
 
 ```bash
 npm install
 ```
 
+### Step 3: Connect with Firebase
+
+In the main `sparkle` folder, copy `.env.example` to a new file `.env.local`, and fill it with the appropriate details, most of which you can find in the Firebase dashboard.
```

Comment:
should we help everyone here saying where the exact needed evn values can be found?